LDAP-primer.md

```diff
@@ -84,7 +85,7 @@ Still keeping it in JSON:
         "userPassword": "secret"
     }
 
-LDAP represents data in a different format, called LDIF (LDAP Data Interchange
+LDAP usually outputs data in a different text format, called LDIF (LDAP Data Interchange
 Format.) Directly translating the above JSON to LDIF gives:
```

Comment:
I think that "usually otputs" undersells LDIF, since a numerous tools accept LDIF _input_, but I agree that "represents" may be misleading, so I'll reformulate that part.